### PR TITLE
Added `open` and `close` events to `amp-lightbox`

### PR DESCRIPTION
**Pull request by inidaname**
This PR exposes lightboxOpen and lightboxClose events.

These events fires after lightbox is fully visible or invisible as requested by @aghassemi, this PR closes Issue #13420

### CODE REVIEW

**Comment by aghassemi** [comment]
@inidaname Thanks for the PR! It looks great. We just need to also document these new events in the actions and events MD file similar to https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md#amp-sidebar and then we can merge!

File: extensions/amp-lightbox/0.1/amp-lightbox.js [hunk 1]
```diff
@@ -15,11 +15,13 @@
  */
 
 import * as st from '../../../src/style';
+import {ActionTrust} from '../../../src/action-trust';
 import {AmpEvents} from '../../../src/amp-events';
 import {CSS} from '../../../build/amp-lightbox-0.1.css';
 import {Gestures} from '../../../src/gesture';
 import {KeyCodes} from '../../../src/utils/key-codes';
 import {Layout} from '../../../src/layout';
+import {createCustomEvent} from '../../../src/event-helper';
```
Comment (aghassemi) [inline extensions/amp-lightbox/0.1/amp-lightbox.js]: please move up for alphabetical sort (the reason the lint is failing on the CI precheck)

Comment (inidaname) [reply extensions/amp-lightbox/0.1/amp-lightbox.js]: Thank You

Comment (cathyxz) [reply extensions/amp-lightbox/0.1/amp-lightbox.js]: You can use `gulp lint --fix` for that too. =)

Comment (inidaname) [reply extensions/amp-lightbox/0.1/amp-lightbox.js]: @cathyxz thank you for that

**Comment by aghassemi** [comment]
@rsimha Regarding Percy failures. This PR can not be causing those changes, not sure what's going on.

**Comment by rsimha** [comment]
@aghassemi This is due to an issue on the Percy side, mitigated in #13589. I've approved the diffs. Feel free to re-run if you'd like.

**Comment by aghassemi** [comment]
I see. Thanks @rsimha

**Pull request merged**